cv32e40s/tests/cfg/pma_test_cfg_4.yaml

```diff
@@ -11,13 +11,13 @@ plusargs:
    +dm_halt_addr=0x32010000
    +dm_exception_addr=0x32010800
    +fix_sp=1
-   #TODO:ropeders un-comment when ISS is patched  +enable_zba_extension=1
-   #TODO:ropeders un-comment when ISS is patched  +enable_zbb_extension=1
-   #TODO:ropeders un-comment when ISS is patched  +enable_zbc_extension=1
-   #TODO:ropeders un-comment when ISS is patched  +enable_zbs_extension=1
+   +enable_zba_extension=1
+   +enable_zbb_extension=1
+   +enable_zbc_extension=1
+   +enable_zbs_extension=1
 riscv_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
 gnu_march:   rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
 corev_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
 llvm_march:  rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
 #TODO:ropeders un-comment when ISS is patched  cv_sw_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
-cv_sw_march: rv32imc
+cv_sw_march: rv32im_zba1p00_zbb1p00_zbc1p00_zbs1p00_zicsr
```

Comment:
c-extension disabled, without zca? is this correct?

Reply:
No, this is an error, I'll fix it.